src/Microsoft.Data.SqlClient/tests/tools/TDS/TDS.Servers/GenericTDSServer.cs

```diff
@@ -544,6 +550,33 @@ protected virtual TDSMessageCollection OnAuthenticationCompleted(ITDSServerSessi
                 responseMessage.Add(featureExtActToken);
             }
 
+            // Check if Json is supported
+            if (session.IsJsonSupportEnabled)
+            {
+                // Create ack data (2 bytes: Version number, Enabled)
+                byte[] data = new byte[2];
+                data[0] = (byte)1;
+                data[1] = (byte)1;
```

Comment:
@deepaksa1  I thought we were going to remove the Enabled byte.
Since you can now test with this TestTds server, it would make sense to change the product code to expect only 1 byte data which is the version and remove data[1]

Reply:
Done